habitat-lab/habitat/core/env.py

```diff
@@ -128,6 +128,28 @@ def __init__(
             }
         )
         self.action_space = self._task.action_space
+
+        # Sharing controller between actions
+        # This should go somewhere else, but it is unclear where
+        # we want to share a controller across actions since the controller
+        # is stateful
+        from habitat_baselines.articulated_agent_controllers import (
+            HumanoidRearrangeController,
+        )
+
+        walk_pose_path = (
```

Comment:
put this code in a helper method to isolate it better.

Reply:
Addressed in latest commit